cmd/wasm/main.go

```diff
@@ -1,51 +1,95 @@
 package main
 
 import (
-	"fmt"
 	"syscall/js"
 
 	"github.com/openmicrotools/jwt-inspect/pkg/jwt"
+	"github.com/openmicrotools/jwt-inspect/pkg/util"
 )
 
+var jsDoc js.Value
+
+func init() {
+	jsDoc = js.Global().Get("document")
+}
+
 func main() {
```

Comment:
**question:** what is the advantage of this jsDoc variable being package scope and populated under init vs being scoped and initialized inside main() for instance? I suppose it avoids needing to make jwtWrapper() a closure but maybe a closure is the better choice.